src/main/java/com/conveyal/datatools/manager/controllers/api/DeploymentController.java

```diff
@@ -249,8 +250,9 @@ private static Deployment createDeploymentFromFeedSource (Request req, Response
 
         if (feedSource.latestVersionId() == null)
             logMessageAndHalt(req, 400, "Cannot create a deployment from a feed source with no versions.");
-
-        Deployment deployment = new Deployment(feedSource);
+        
+        boolean useDefaultRouter = !isExtensionEnabled("nysdot");
```

Comment:
```suggestion
        // NYSDOT deployments do not use the default router for feed-specific deployments 
        // because they are expected to use the test server, which allows for building graphs over the wire.
        boolean useDefaultRouter = !isExtensionEnabled("nysdot");
```